node/test/toolrunnertests.ts

```diff
@@ -409,21 +469,25 @@ describe('Toolrunner Tests', function () {
         };
 
         if (os.platform() === 'win32') {
-            var find = tl.tool(tl.which('FIND', true))
-                .arg('System Idle Process');
-
-            var tasklist = tl.tool(tl.which('tasklist', true));
-            tasklist.pipeExecOutputToTool(find);
+            var matchExe = tl.tool(compileMatchExe())
+                .arg('0') // exit code
+                .arg('line 2'); // match value
+            var outputExe = tl.tool(compileOutputExe())
+                .arg('0') // exit code
+                .arg('line 1')
+                .arg('line 2')
+                .arg('line 3');
+            outputExe.pipeExecOutputToTool(matchExe);
```

Comment:
the match program i wrote waits for STDIN to close. i think find.exe may have been too hasty for how we were trying to test here. it was previously flaky, and then reliably failing when i switched to hosted. the custom program appears to be reliably working.